server/routes/api.py

```diff
@@ -82,7 +82,7 @@ def single_author(author_id: int) -> Response:
 
 @bp.route("/authors/<int:author_id>/posts/<int:post_id>", methods=["GET"])
 def get_post(author_id: int, post_id: int) -> Response:
```

Comment:
should we restrict the access to this post for only users logged in?

Reply:
It's only returning public posts so I don't believe we need authentication right now.

Reply:
makes sense